cirq/ops/gate_features.py

```diff
@@ -31,11 +31,8 @@ def qubit_index_to_equivalence_group_key(self, index: int) -> int:
         return 0
 
 
-class SingleQubitGate(raw_types.Gate, metaclass=abc.ABCMeta):
-    """A gate that must be applied to exactly one qubit."""
-
-    def _num_qubits_(self) -> int:
-        return 1
+class SometimesOneQubitGate(raw_types.Gate, metaclass=abc.ABCMeta):
```

Comment:
can you rename this to `SupportsOnEach`?  And then we can merge it!

Reply:
Sorry, I had forgotten to git push. Like this?